hack/ci-e2e.sh

```diff
@@ -1,10 +1,21 @@
 #!/usr/bin/env bash
 
+# -----------------------------------------------------------------------------
+# Description: This script sets up the environment and runs E2E tests for the
+#              BMO project. It uses either vbmc or sushy-tools based on 
+#              the USE_TOOL environment variable.
+# Usage:       export USE_TOOL="vbmc"  # Or "sushy-tools"
+#              ./ci-e2e.sh
+# -----------------------------------------------------------------------------
+
 set -eux
 
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 cd "${REPO_ROOT}" || exit 1
 
+# USE_TOOL can be set to either "vbmc" or "sushy-tools"
+USE_TOOL=${USE_TOOL:-"vbmc"}  # Defaults to vbmc if not set
```

Comment:
```suggestion
USE_TOOL=${USE_TOOL:-"vbmc"}
```
Comment is only stating the obvious, better not have it.